components/layout/block.rs

```diff
@@ -1491,7 +1491,7 @@ impl BlockFlow {
         debug_assert!(!self.is_inline_flex_item());
 
         // Compute the available space for us, based on the actual floats.
-        let rect = self.base.floats.available_rect(Au(0),
```

Comment:
Just curious, is it this change or the `force_reflow` stuff what makes the new tests pass?